COMPONENT_MATURITY.md

```diff
@@ -0,0 +1,170 @@
+# Component maturity
+
+Each component is tested against the following items to guage the component's maturity. Use this list to understand how to test the component and when to make a checklist item 'Not applicable'.
+
+## Accessibility
+### WCAG 2.1 Level AA Conformance
+All Axe and Storybook plugins for WCAG AA compliance have passed.
+#### About
+Code is checked with the [Axe Core plugin (external link)](https://github.com/dequelabs/axe-core "Axe core plugin repository") that finds on average 57% of WCAG issues automatically. You can see the complete list of rules that are checked in the [rules descriptions (external link)](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md "Axe core plugin rules description"). 
+
+You can use Storybook to test for WCAG violations and color blindness filters.
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+Run `yarn test:a11y`
+
+### Supports Forced Colors Mode (FCM)
+While using FCM the components text is legible and improves readability.
+
+#### About
+In Forced Color Mode (FCM) in Windows, verify in Edge, Chrome, and Firefox that:
+- All background colors are reset to a single color.
+- All elements that need a border have one.
+- All SVG elements and icons have a contrast color applied.
+- Color is not the only means used to convey meaning with elements.
+- Focus styles are clearly indicated.
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+The full suite of visual regression tests takes screenshots of what each component looks like in forced-colors mode. One can also [turn on forced-colors mode in browser developer tools (external link)](https://devtoolstips.org/tips/en/emulate-forced-colors/ "Turn on forced-colors mode"). They should be reviewed by the team to determine if the component meets the above criteria.
+
+### Color
+Meets AA color contrast standards for accessibility and color blindness.
+
+#### About
+Code is verified for color contrast using the Axe Core plugin.
+
+Color blindness is tested using a color blindness simulator and visually inspected for:
+- Elements with low color contrast
+- Text readability
+- Link visibility
+- Errors and warnings that might rely on color
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+- Code - Run `yarn test:a11y`
+- Design - Designs are verified using a plugin within Sketch to check color contrast and color blindness.
+
+### Screen readers
+VoiceOver, NVDA, and JAWS were used to test for concise communication and interaction.
+
+#### About
+Components are tested in the following screen readers: 
+- Mac OS
+-- Safari + VoiceOver
+- Windows
+-- NVDA + Edge
+-- JAWS + Edge
+-- iOS + Safari + Mobile VoiceOver (as time permits)
+
+#### When to define as 'Not applicable'?
+When the component has no interactive elements like a button or text input a 'Not applicable' state can be applied.
```

Comment:
I actually wonder if this leaves out a lot of components that we do want tested for how they appear to a screen reader. One example would be the `Alert`. It's not interactive, but it has an `aria-labelledby` attribute and some screen-reader-only text (prefixes to headings like "Warning: ") and optional auto-focusing behavior. Even without the auto-focusing behavior, I would want to know that the Alert experience is good for screen readers.

Reply:
This is a great point! There are alot of elements this situation could apply to. @SparrowSpit Would there be any instance where Voiceover testing wouldn't be applicable to the component?